flatnav_python/python_bindings.cpp

```diff
@@ -168,13 +168,6 @@ class PyIndex : public std::enable_shared_from_this<PyIndex<dist_t, label_t>> {
         /* ef_search = */ ef_search,
         /* num_initializations = */ num_initializations);
 
-    if (top_k.size() != K) {
```

Comment:
Relaxing this condition here until I figure out why beamSearch sometimes returns exactly one candidate neighbor here: https://github.com/BlaiseMuhirwa/flatnav-experimental/blob/f90b479c23ce59daf17c45295876286c8283076d/flatnav/Index.h#L311